int-swap-dApp/src/js/app.js

```diff
@@ -369,10 +431,12 @@ App = {
 
 
       // have proposal owner deposit escrow into Int Swap contract using msg.data and msg.value
-      return setTimeout(function(){
-        intSwapInstance.proposerDepositIntoEscrow(wei_escrow_amount, escrow_percent, { from: $proposal_owner_address.val(), gas: 3000000, value: wei_escrow_amount });
-      }, 1000);
-      
+      // return setTimeout(function(){
+      //   intSwapInstance.proposerDepositIntoEscrow(wei_escrow_amount, escrow_percent, { from: $proposal_owner_address.val(), gas: 3000000, value: wei_escrow_amount });
+      // }, 1000);
+
+      return intSwapInstance.proposerDepositIntoEscrow(wei_escrow_amount, escrow_percent, { from: $proposal_owner_address.val(), gas: 5000000, value: wei_escrow_amount });
+
```

Comment:
@simonnguyen3054 Did increasing the gas amount help? Because if it doesn't make a difference we should change it back to 3000000.